_search-plugins/search-pipelines/script-processor.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Script processor
+nav_order: 30
+has_children: false
+parent: Search pipelines
+grand_parent: Search
+---
+
+# Script processor
+
+Script processor is part of search pipeline functionality. Search pipelines is an experimental feature. For updates on the progress of search pipelines, or if you want to leave feedback that could help improve the feature, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6278).    
```

Comment:
```suggestion
The script processor is part of the functionality of a search pipeline, which is an experimental feature. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6278).    
```

Reply:
> The script processor is part of the functionality of a search pipeline

That's confusing.

A script processor is a type of processor that may be added to a search pipeline, if you want.

Reply:
@msfroh I have put the same experimental warning on all files so they are uniform.